uwrt_mars_rover_drivetrain/uwrt_mars_rover_drivetrain_description/urdf/drivetrain.macro.xacro

```diff
@@ -120,4 +120,164 @@
     </ros2_control>
   </xacro:macro>
 
+  <xacro:macro name="nav_imu">
+    <link name="nav_imu_link">
+      <visual>
+        <geometry>
+          <box size="0.1 0.1 0.1"/>
+        </geometry>
+        <!-- <material name="cyan"> -->
+        <!--   <color rgba"0 1.0 1.0 1.0"/> -->
+        <!-- </material> -->
+      </visual>
+
+      <collision>
+        <geometry>
+          <box size="0.1 0.1 0.1"/>
+        </geometry>
+      </collision>
+
+      <!-- <xacro:box_inertia m="0.1" w="0.1" d="0.1" h="0.1"/> -->
+    </link>
+
+    <joint name="nav_imu_joint" type="fixed">
+      <parent link="base_link"/>
+      <child link="nav_imu_link"/>
+      <origin xyz="0 0 0.01"/>
+    </joint>
+
+    <gazebo reference="nav_imu_link">
+      <sensor name="nav_imu_sensor" type="imu">
+       <plugin filename="libgazebo_ros_imu_sensor.so" name="imu_plugin">
+          <ros>
+            <namespace>/demo</namespace>
```

Comment:
Is this the right namespace we want to be using for the IMU? Demo seems vague to me